prereise/gather/griddata/hifld/model.py

```diff
@@ -0,0 +1,29 @@
+from prereise.gather.griddata.hifld.data_access.load import load_csv
+from prereise.gather.griddata.hifld.calculate.remap import (
+    get_zone_mapping,
+    get_sub_mapping,
+    get_line_mapping,
+)
+from prereise.gather.griddata.hifld.calculate.clean import clean_substations
+
+
+def get_grid_data(e_csv, t_csv, z_csv):
+    """Load HIFLD data and create dataframes corresponding to the input csv's
+    for the BES grid.
+
+    :param str e_csv: path of the HIFLD substation csv file
+    :param str t_csv: path of the HIFLD transmission csv file
+    :param str z_csv: path of the zone csv file
+    """
+
+    zone_data = load_csv(z_csv)
+    zone_dic, zone_interconnect_dic = get_zone_mapping(zone_data)
```

Comment:
can we use `zone2id` as a variable name here? This is what we use in **PowerSimData**

Reply:
`zone_interconnect_dic` is not used I believe so we do `zone_id, _ = get_zone_mapping(zone_data)` or just `zone_id = get_zone_mapping(zone_data)` if we don't return the second element (see comment under `get_zone_mapping` function)

Reply:
Or just `zone_id = pd.Series(zone_data.zone_id, index=zone_data.zone_name").to_dict()`, and avoid defining the function at all.

Reply:
Or really, since the current `zone_dic` is only used to get an interable for use within `clean_substations`, we just need `zone_data.zone_id`.